src/thrift/parquet.thrift

```diff
@@ -203,6 +209,8 @@ struct Statistics {
    3: optional i64 null_count;
    /** count of distinct values occurring */
    4: optional i64 distinct_count;
+   5: optional bool bloom_filter_enabled;
```

Comment:
Why have an extra boolean? Why not just check whether blomm_filter_stats exists?